pkg/destroy/openstack/openstack.go

```diff
@@ -99,9 +99,9 @@ func (o *ClusterUninstaller) Run() error {
 
 func deleteRunner(deleteFuncName string, dFunction deleteFunc, opts *clientconfig.ClientOpts, filter Filter, logger logrus.FieldLogger, channel chan string) {
 	backoffSettings := wait.Backoff{
-		Duration: time.Second * 10,
+		Duration: time.Second * 15,
```

Comment:
This isn't completely obvious to me why we're making the initial backoff duration longer. Could you explain?

Reply:
It's not necessary, but there were two reasons:
1. Normally it takes more than 10 seconds to destroy an instance, so increasing the initial timeout from 10 to 15 seconds allows to reduce the number of requests, and therefore ResourceNotFound exceptions.
2. It sets the overall timeout to 10 minutes, but with 10 seconds it is just 7. 10 minutes looks prettier, imho :)